modules/oke/variables.tf

```diff
@@ -47,6 +47,8 @@ variable "use_encryption" {
 
 variable "kms_key_id" {}
 
+variable "volume_kms_key_id" {}
```

Comment:
As above, please prefix with node pool. Can you also raise a new issue to rename the existing variable `kms_key_id` to `cluster_kms_key_id` so we can distinguish between the 2?